src/types/gasPriceOracle.ts

```diff
@@ -0,0 +1,50 @@
+import type {
+  Abi,
+  BlockTag,
+  EncodeFunctionDataParameters,
+  PublicClient,
+  TransactionSerializableEIP1559,
+  Transport,
+} from 'viem'
+import type { Chain } from 'viem/chains'
+
+/**
+ * Options to query a specific block
+ */
+type BlockOptions = {
```

Comment:
why not export this and use it in the other files instead of re stating it? 🤔

Reply:
Can do this we just landed here from a refactor to make it one function per file

Reply:
Roger yeah appreciate you breaking out, just would prefer to DRY up types if we can.